testutils/src/index.ts

```diff
@@ -193,24 +193,43 @@ export function createFileContext(
 }
 
 /**
- * Create a context for a notebook.
+ * Create and initialize context for a notebook.
  */
-export async function createNotebookContext(
-  path?: string,
-  manager?: ServiceManager.IManager
+export async function initNotebookContext(
+  options: {
+    path?: string;
+    manager?: ServiceManager.IManager;
+    startKernel?: boolean;
+  } = {}
 ): Promise<Context<INotebookModel>> {
   const factory = Private.notebookFactory;
 
-  manager = manager || Private.getManager();
-  path = path || UUID.uuid4() + '.ipynb';
+  const manager = options.manager || Private.getManager();
+  const path = options.path || UUID.uuid4() + '.ipynb';
+  const startKernel =
+    options.startKernel === undefined ? false : options.startKernel;
```

Comment:
No need to actually take this comment, but the same thing can be done a bit more tersely with `startKernel = !!options.startKernel`

Reply:
True, and I thought about it, but I actually like this syntax better. More verbose, but much clearer about the intended default value.

Reply:
Yep, makes sense.